Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -1076,9 +1075,9 @@ private async Task PerformLibraryValidation(IProgress<double> progress, Cancella
 
             var innerProgress = new ActionableProgress<double>();
 
-            innerProgress.RegisterAction(pct => progress.Report(pct * .96));
+            innerProgress.RegisterAction(pct => progress.Report(pct * pct * 0.96));
```

Comment:
@dkanada This looks very weird to me. Why `pct * pct`?

Reply:
Square the value so percentage doesn't progress as quickly in the beginning.

Reply:
Squaring the value makes it progress quicker though?

Reply:
Yes, this would explain the issue. I thought progress was reported as a decimal value, not an integer value.

Reply:
Decimal, integer. Does it matter? Squaring it will always make it bigger.

Reply:
A decimal between zero and one will shrink when you square it.

Reply:
> I thought progress was reported as a decimal value, not an integer value.
> A decimal between zero and one will shrink when you square it.

A decimal can also be larger eg. 6.4, but I get what you mean now.